twoliter/embedded/rpm2kit

```diff
@@ -30,3 +30,6 @@ done
 
 createrepo_c "${KIT_DIR}"
 dnf --disablerepo '*' --repofrompath "kit,file:///${KIT_DIR}" repoquery --all
+
+BASE_NAME=$(cat $OUTPUT_DIR/kit-metadata.json | jq '. | .kit + "-" + .version')
```

Comment:
Where does this magic `kit-metdata.json` file come from?

In general you should align your output artifacts around the same sort of naming done in `rpm2img` and `imghelper`:
```
FILENAME_PREFIX="${KIT_NAME:?}--${ARCH:?}-${VERSION_ID:?}-${BUILD_ID:?}"
```

And these fields should be used internally as well where it makes sense, not just applied to the name of the tar archive.